model/converter/thrift/zipkin/fixtures/jaeger_01.json

```diff
@@ -61,7 +61,7 @@
       }
     },
     {
-      "traceID": "1",
+      "traceID": "20000000000000001",
```

Comment:
checkout https://github.com/uber/jaeger/blob/5e5a361babba385bd0b9868bace1e10c75fc0829/model/span.go#L173 it overwrites the unmarshalling logic so that it can take a 128bit hex string and convert it into high and low